src/main/scala/org/broadinstitute/monster/etl/encode/ExtractionPipeline.scala

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.monster.etl.encode
```

Comment:
Nitpick: could we use `org.broadinstitute.monster.encode.extraction` as the package? I've been using `org.broadinstitute.monster.clinvar` as the package in the ClinVar code migration.